VTEX - Audience API.json

```diff
@@ -80,7 +83,7 @@
                         }
                     },
                     "400": {
-                        "description": "Bad Request",
+                        "description": "Bad Request - Occurs when the request body is empty or missing properties, or if the account is not identified from the context or query parameters.",
```

Comment:
O campo "description" dentro de "400" quando renderizado no portal não teria uma visualização boa para uma quantidade tão grande de conteúdo, ele é pensado para o nome do status code do erro só ("OK", "Bad request", etc.), como no print. 

Minha sugestão: transferir a explicação do erro para a "description" dentro do "schema" da response. Exemplo:


"400": {
        "description": "Bad Request",
        "content": {
            "application/json": {
                "schema": {
                    "type": "object",
                    "properties": {
                        "Message": {
                            "description": "Occurs when the request body is empty, there are missing properties, or the account could not be found from the context or query parameters.",
                            "type": "string"
                        }
                    }
                },
                "example": {
                    "Message": "Bad request"
                }
            }
        }
    }

![image](https://github.com/user-attachments/assets/bd84e611-fcd1-4032-b695-1fb50c6ebccf)

Reply:
OLá, @Isabella-Veloso!
Eu acredito que o local mais apropriado para incluir essa informação seria logo após o "Bad Request", pois isso oferece mais contexto sobre a situação em que ele ocorre, sem necessariamente confundir com o tipo de mensagem retornada. Além disso, consultei o DevPortal, e verifiquei que existem outras [APIs](https://developers.vtex.com/docs/api-reference/license-manager-api#put-/api/license-manager/users/-userId-/roles?endpoint=put-/api/license-manager/users/-userId-/roles) que possuem descrições adicionais ao "Bad Request" e que o portal realizou a renderização de forma correta.

Reply:
Oi Geise, levei a conversa para o Slack, para incluir outras pessoas, já que estamos falando sobre padrão de openAPI schema: 

https://vtex.slack.com/archives/C01NX6BUM19/p1726159709899769?thread_ts=1726138857.101059&cid=C01NX6BUM19